webpack_loader/utils.py

```diff
@@ -124,3 +124,10 @@ def get_static(asset_name, config='DEFAULT'):
         public_path = getattr(settings, 'STATIC_URL')
 
     return '{0}{1}'.format(public_path, asset_name)
+
+def get_asset(asset_name, config='DEFAULT'):
```

Comment:
Please add a docstring similar to the one from `get_static`, but clarify why this is different.